litmus-portal/frontend/src/views/AnalyticsDashboard/MonitoringDashboardPage/InfoDropdown/styles.ts

```diff
@@ -0,0 +1,89 @@
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '100%',
+  },
+  header: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    background: theme.palette.disabledBackground,
+  },
+  headerText: {
+    fontWeight: 'bold',
+    fontSize: '1rem',
+    fontHeight: '140%',
```

Comment:
fontHeight should be in rem or px? 140%?

Reply:
yes updating this

Reply:
This can be ignored